src/frontend/screens/Onboarding/walletSelection/screens/info/index.tsx

```diff
@@ -28,66 +31,84 @@ const InfoText = function ({ icon, title, details }: InfoTextProps) {
   )
 }
 
-interface WalletInfoScreenProps {
-  skipClicked: () => void
-  createWalletClicked: () => void
-  mmInitialized: boolean
-}
-
 const WalletInfoScreen = (props: WalletInfoScreenProps) => {
   return (
-    <div className={WalletInfoStyles.walletInfoScreen}>
-      <div className={`title ${WalletInfoStyles.title}`}>
-        {t(
-          'hyperplay.onboarding.walletSelection.screens.info.title',
-          `What is a wallet?`
-        )}
-      </div>
-      <InfoText
-        icon={WalletOptionIcon}
-        title={t(
-          'hyperplay.onboarding.walletSelection.screens.info.digitalAssets.title',
-          `A Home for your Digital Assets`
-        )}
-        details={t(
-          'hyperplay.onboarding.walletSelection.screens.info.digitalAssets.details',
-          `Wallets are used to send, receive, store, and display digital assets like tokens and NFTs.`
-        )}
-      />
-      <InfoText
-        icon={WalletIcon}
-        title={t(
-          'hyperplay.onboarding.walletSelection.screens.info.login.title',
-          `A New Way to Log In`
-        )}
-        details={t(
-          'hyperplay.onboarding.walletSelection.screens.info.login.details',
-          `An alternative to creating new accounts and passwords on every website, just connect your wallet instead.`
-        )}
-      />
-      <div className={`caption-sm ${WalletInfoStyles.caption}`}>
-        {t(
-          'hyperplay.onboarding.walletSelection.screens.info.createWalletDescription',
-          `Click in “Create a wallet” below if you do not have a wallet or want to create a new one:`
-        )}
-      </div>
-      <div className={WalletInfoStyles.actionButtonContainer}>
-        <Button type="tertiary" onClick={props.skipClicked}>
+    <>
+      <div className={WalletInfoStyles.walletInfoScreen}>
+        <div className={WalletInfoStyles.connectWalletSvgContainer}>
+          <span className={WalletInfoStyles.blurBackground}></span>
+          <ConnectWallet className={WalletInfoStyles.walletSvg} />
+        </div>
+        <div className={`title ${WalletInfoStyles.title}`}>
+          {t(
+            'hyperplay.onboarding.walletSelection.screens.newInfo.title',
+            `Don't Just Play-Get Rewarded`
+          )}
+        </div>
+        <div className={`body-sm ${WalletInfoStyles.smallTitle}`}>
           {t(
-            'hyperplay.onboarding.walletSelection.screens.info.skip',
-            `Skip for now`
+            'hyperplay.onboarding.walletSelection.screens.info.digitalAssets.smallTitle',
+            `Connect your wallet to activate powerful features:`
           )}
-        </Button>
-        {!props.mmInitialized ? (
-          <Button onClick={props.createWalletClicked}>
+        </div>
+        <div className={`body-sm ${WalletInfoStyles.description}`}>
```

Comment:
looks like `body-sm` is 14px in figma and 16px in hp/ui. we should fix that. I think it is one of the reasons the height of this section is larger than the figma design